### PR TITLE
Package Requirements

### PR DESCRIPTION
**Pull request by aspeake**
New configuration file; this is the first step in organizing Scout as a Python package for easier development and better reproducibility of results. This PR allows requirements to be installed with `pip install .`

### CODE REVIEW

**Comment by aspeake** [comment]
@trynthink rebased after the `profiler` merge, should be ready for your review

File: pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+readme = "README.md"
+version = "0.9"
+authors = [
+  {email = "chioke.harris@nrel.gov"},
+  {name = "Chioke Harris"},
+  {email = "jared.langevin@lbl.gov"},
+  {name = "Jared Langevin"}
+]
+requires-python = ">=3.8"
+license = {text = "Apache-2.0"}
```
Comment (trynthink) [inline pyproject.toml:21]: Change the license reference to [LICENSE.md](https://github.com/trynthink/scout/blob/master/LICENSE.md).

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+dependencies = [
+    "pandas >= 2.0",
```
Comment (trynthink) [inline pyproject.toml:23]: I'm not sure if we have a requirement for v2.2 versus v2.0, but v2.2 was the previously stated version requirement.

Comment (aspeake) [reply pyproject.toml:23]: The most recent pandas release was v2.1.1 though...

Comment (trynthink) [reply pyproject.toml:23]: In that case...v2.0 is good.

**Review by trynthink — APPROVED** [review]

**Pull request merged**